sdk/communication/Azure.Communication.CallingServer/tests/CallAutomationClient/CallAutomationClientAutomatedLiveTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Communication.Identity;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Microsoft.AspNetCore.Http;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Azure.Communication.CallingServer
+{
+    internal class CallAutomationClientAutomatedLiveTests : CallAutomationClientAutomatedLiveTestsBase
+    {
+        public CallAutomationClientAutomatedLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [Test]
+        public async Task CreateCallToACSGetCallAndHangUpCallTest()
+        {
+            /* Test case: ACS to ACS call
+             * 1. create a CallAutomationClient.
+             * 2. create a call from source to one ACS target.
+             * 3. get updated call properties and check for the connected state.
+             * 4. hang up the call.
+             * 5. once call is hung up, verify disconnected event
+            */
+            if (SkipCallingServerInteractionLiveTests)
```

Comment:
nit: change SkipCallingServerInteractionLiveTests to SkipCallAutomationInteractionLiveTests